### PR TITLE
visual enhancements to ProfileGoalsSection component

### PR DESCRIPTION
**Pull request by kwiedenhaupt**
1.  added padding to the top
2.  sliced the array so that only the first five goals map on the screen
Currently it displays the first five goals.  It would be possible to do the most recent, but it would make sense that the oldest ones should be addressed first I would think?

### CODE REVIEW

**Comment by kwiedenhaupt** [comment]
Screen Shot 2022-10-21 at 9 47 12 PM

**Comment by kwiedenhaupt** [comment]
Screen Shot 2022-10-21 at 9 58 27 PM

This is the total number of goals I have: where only five are showing on the profile page.

File: components/coding/ProfileGoalsSection.tsx [hunk 1]
```diff
@@ -24,6 +24,7 @@ export default function ProfileGoalsSection({ user }: GoalsSectionProps) {
       },
     }
   );
+  const slicedUserGoals = userGoals.slice(0, 5);
```
Comment (iamnotreddy) [inline components/coding/ProfileGoalsSection.tsx:27]: one small change I'm thinking here - how about we filter to only show current goals, but keep the slice so we show 5 current goals at most? 

I think this makes more sense from a user perspective - what do you think?

**Review by iamnotreddy — APPROVED** [review]
Check my one comment but I'm approving

**Pull request merged**